eng/ci/code-mirror.yml

```diff
@@ -4,6 +4,7 @@ trigger:
     # Keep this set limited as appropriate (don't mirror individual user branches).
     - main
     - release/*
+    - feature/*
```

Comment:
Do we need official builds of feature branches? If so, we need to also lock down creation of these branches.

If we just want CI validation of feature branches, we can look into using unofficial 1ES PT builds for that in our public org.

Reply:
That's a good point, we probably don't need that and can just use the unofficial build